### PR TITLE
Preferences, Help Shortcuts

### LINKED ISSUE #3464: Add "Help" keyboard shortcut and link it to the help-button URL, when possible
The keyboard shortcut should be `F1` on Linux and Windows, and `⌘?` on macOS.

It should perform the same action as clicking on the help button.

### PR DESCRIPTION
**Pull request by torchiaf**
Fixes #3464

Signed-off-by: Francesco Torchia <francesco.torchia@suse.com>

### CODE REVIEW

File: pkg/rancher-desktop/typings/electron-ipc.d.ts [hunk 1]
```diff
@@ -77,6 +77,8 @@ export interface IpcMainEvents {
 
   /** Only for the preferences window */
   'preferences/load': () => void;
+
+  'help/preferences/open-url': () => void;
```
Comment (rak-phillip) [inline pkg/rancher-desktop/typings/electron-ipc.d.ts:81]: Would `preferences/help` be more consistent with the existing `preferences` prefix, or do we anticipate to have many `help` events?

**Review by rak-phillip — COMMENTED** [review]
I think this looks good and functions as advertised. I only have a few minor thoughts that came up during review.

File: pkg/rancher-desktop/typings/electron-ipc.d.ts (hunk 1 above)
Comment (torchiaf) [reply pkg/rancher-desktop/typings/electron-ipc.d.ts:81]: We will implement the other events coming from Main window.

**Comment by torchiaf** [comment]
Thanks @rak-phillip , updated :+1:

**Review by rak-phillip — APPROVED** [review]
LGTM

**Pull request merged**